back/database/migrations/2020_04_26_140003_create_messages_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateMessagesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('messages', function (Blueprint $table) {
+            $table->bigIncrements('id');
```

Comment:
Je préfère qu'on fasse comme dans les autres migrations ici donc qu'on utilise `$table->increments('id')` à la place, histoire que ça reste cohérent avec le reste du code
```suggestion
            $table->increments('id');
```